ios/FluentUI.Demo/FluentUI.Demo/Demos/NotificationViewDemoController_SwiftUI.swift

```diff
@@ -51,22 +52,23 @@ struct NotificationDemoView: View {
         let messageButtonAction = hasMessageAction ? { showAlert = true } : nil
         let hasMessage = !message.isEmpty
         let hasTitle = !title.isEmpty
-        let notification = FluentNotification(style: style,
-                                              isFlexibleWidthToast: isFlexibleWidthToast,
-                                              message: hasMessage ? message : nil,
-                                              attributedMessage: hasAttribute && hasMessage ? attributedMessage : nil,
-                                              title: hasTitle ? title : nil,
-                                              attributedTitle: hasAttribute && hasTitle ? attributedTitle : nil,
-                                              image: image,
-                                              trailingImage: trailingImage,
-                                              trailingImageAccessibilityLabel: showTrailingImage ? "Circle" : nil,
-                                              actionButtonTitle: actionButtonTitle,
-                                              actionButtonAction: actionButtonAction,
-                                              messageButtonAction: messageButtonAction)
-                            .overrideTokens(overrideTokens ? NotificationOverrideTokens() : nil)
 
         VStack {
-            notification
+            Rectangle()
+                .foregroundColor(.clear)
+                .presentNotification(style: style,
+                                     isFlexibleWidthToast: $isFlexibleWidthToast.wrappedValue,
+                                     message: hasMessage ? message : nil,
+                                     attributedMessage: hasAttribute && hasMessage ? attributedMessage : nil,
+                                     isBlocking: false,
+                                     isPresented: .constant(true),
+                                     title: hasTitle ? title : nil,
+                                     attributedTitle: hasAttribute && hasTitle ? attributedTitle : nil,
+                                     image: image,
```

Comment:
What happened to trailingImage and trailingImageAccessibilityLabel?

Reply:
I made a mistake and have to PRs out at the same time, but I didn't realize when originally adding the trialingImage that I needed to update the `.presentNotification` modifier, so they're being updated in #1151

Reply:
Moved those changes to this PR instead.